projects/rocprofiler-sdk/source/bin/rocprofv3-avail.py

```diff
@@ -312,20 +330,26 @@ def print_agent_counter(counters):
             print("\n")
 
 
-def info_pmc(args):
+def info_pmc(args, spm=False):
     from rocprofv3 import avail
 
-    agent_counters = avail.get_counters()
+    _args = None
+    if spm:
+        agent_counters = avail.get_spm_counters()
+        _args = args.spm
+    else:
+        agent_counters = avail.get_counters()
+        _args = args.pmc
     agent_info_map = avail.get_agent_info_map()
 
-    def print_pmc_info(args, pmc_counters):
+    def print_pmc_info(_args, pmc_counters):
 
-        if not args.pmc:
+        if not _args:
```

Comment:
In case `_args` is boolean, then this is perfect. Otherwise, we might consider 
```suggestion
        if _args is None:
```

Then we can avoid implicit casting to boolean in Python.